### PR TITLE
User view scaffolding

### PR DESCRIPTION
**Pull request by Fijo-Lurk**
- added html boilerplate
- responsive layout
- installed font-awesome

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -3,9 +3,19 @@ import ReactDOM from 'react-dom';
...
 import { Main } from './layout';
 import ContextProvider from './provider/ContextProvider';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+import {
+	faStar,
+	faMapPin,
+	faPhone,
+	faEnvelope,
+} from '@fortawesome/free-solid-svg-icons';
```
Comment (axelra82) [inline frontend/src/index.js:13]: Duplicate of @fortawesome/free-solid-svg-icons

File: frontend/src/pages/user/style.scss [hunk 2]
```diff
@@ -0,0 +1,104 @@
+
+h1,h2,h3,h4,p{
```
Comment (axelra82) [inline frontend/src/pages/user/style.scss]: Don't make it global 😉

**Review by axelra82 — CHANGES_REQUESTED** [review]
Check comments

File: frontend/src/index.js (hunk 1 above)
Comment (Fijo-Lurk) [reply frontend/src/index.js:13]: Done!

File: frontend/src/pages/user/style.scss (hunk 2 above)
Comment (Fijo-Lurk) [reply frontend/src/pages/user/style.scss]: done!

**Comment by Fijo-Lurk** [comment]
I have fix the comments you ask for.

**Review by Fijo-Lurk — COMMENTED** [review]
I have fix the comments you ask for.

**Comment by natalienilsson93** [comment]
Merging LGTM.

**Pull request merged**